docs/concepts/storage/persistent-volumes.md

```diff
@@ -661,6 +664,9 @@ You can see [vSphere example](https://git.k8s.io/kubernetes/examples/volumes/vsp
   ```
   $ kubectl create secret generic ceph-secret --type="kubernetes.io/rbd" --from-literal=key='QVFEQ1pMdFhPUnQrSmhBQUFYaERWNHJsZ3BsMmNjcDR6RFZST0E9PQ==' --namespace=kube-system
   ```
+* `fsType`: fsType that are supported by kubernetes. Default: `"ext4"`.
+* `imageFormat`: Ceph RBD image format, "1" or "2". Default is "1".
+* `imageFeatures`: Ceph RBD image format 2 features, comma delimited. This parameter is optional, and only be used if you set `imageFormat` to "2". Currently supported features are `layering` only. Default is "", and no features is turned on.
```

Comment:
suggested edit:

This parameter is optional and `should` only be used if you set \`imageFormat\` to "2". Currently supported features are \`layering\` only. Default is "", and no features `are` turned on.